setup.cfg

```diff
@@ -38,7 +38,7 @@ zip_safe = false
 tests =
     pytest
     pytest-asyncio
-    mypy>=0.800
+    mypy @ git+https://github.com/python/mypy.git@master
```

Comment:
```suggestion
    mypy >= 0.920
```

Reply:
0.920 was released yesterday http://mypy-lang.blogspot.com/2021/12/mypy-0920-released.html and supports ignoring ParamSpec in Callable[P, R] position